main.go

```diff
@@ -183,8 +186,8 @@ func main() {
 
 	router := route.New()
 
-	RegisterWeb(router)
-	api.Register(router.WithPrefix("/api"))
+	RegisterWeb(router.WithPrefix(amURL.Path))
+	api.Register(router.WithPrefix(amURL.Path + "/api"))
```

Comment:
A full URL is not a path. Of course that doesn't work and it shouldn't. But this is a path (`url.URL.Path`) :)

Do as you wish. But it would be correct to apply here and prevent keep things working, when sanitization somewhere up the chain changes.